config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java

```diff
@@ -382,4 +388,229 @@ public GroupkeyListenserStatus getListeners(HttpServletRequest request, HttpServ
             return rr;
         }
     }
+
+    @RequestMapping(params = "export=true", method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<byte[]> exportConfig(HttpServletRequest request,
+                                               HttpServletResponse response,
+                                               @RequestParam("group") String group,
+                                               @RequestParam(value = "appName", required = false) String appName,
+                                               @RequestParam(value = "tenant", required = false,
+                                                   defaultValue = StringUtils.EMPTY) String tenant,
+                                               @RequestParam(value = "ids", required = false)List<Long> ids) {
+        String idsStr = idList2String(ids);
```

Comment:
plz use this tool class which is `com.google.common.base.Joiner`  to complete the collection conversion 
``` java
// String idsStr = idList2String(ids);
String idsStr = Joiner.on(",").join(ids);
```